etna/models/linear.py

```diff
@@ -1,11 +1,51 @@
+import numpy as np
+import pandas as pd
 from sklearn.linear_model import ElasticNet
 from sklearn.linear_model import LinearRegression
 
-from etna.models.sklearn import SklearnMultiSegmentModel
-from etna.models.sklearn import SklearnPerSegmentModel
+from etna.models.base import NonPredictionIntervalContextIgnorantAbstractModel
+from etna.models.mixins import MultiSegmentModelMixin
+from etna.models.mixins import NonPredictionIntervalContextIgnorantModelMixin
+from etna.models.mixins import PerSegmentModelMixin
+from etna.models.sklearn import _SklearnAdapter
 
 
-class LinearPerSegmentModel(SklearnPerSegmentModel):
+class _LinearAdapter(_SklearnAdapter):
```

Comment:
Have we wrote anywhere how exactly components are created for linear models? How the user should guess about it?

Reply:
No we haven't))
We haven't done it for any of the models, may be it is better do add notes about it in the models docstring in the separate issue

Reply:
Add Notes here